include/effectmanager.h

```diff
@@ -153,9 +153,19 @@ class  EffectManager : public IJSONSerializable
         ClearEffects();
     }
 
-    std::shared_ptr<GFXBase> GetBaseGraphics()
+    // SetTempEffect - Sets a temporary effect to be played until remote changes it.
+    //                 The effect must have already had its Init() function called.
+
+    void SetTempEffect(std::shared_ptr<LEDStripEffect> effect)
+    {
+        _tempEffect = effect;
+    }
+

```

Comment:
I could swear this functionality existed for the atomic lamp already, but could not find a way to actually set the temp effect, so added this method.  Let me know if I missed something that would have obviated the need for it!

Reply:
As far as I can remember, the `_tempEffect` could only be set through the constructor the whole time. But there is a theoretical chance that the possibility did exist a long time ago, and I removed it because the function wasn't used anywhere - I have done some dead code clean-up over time.